ClosedTypeHierarchyDiagnosticSuppressor/SwitchExpressionSuppressor.cs

```diff
@@ -13,6 +13,15 @@ namespace SvSoft.Analyzers.ClosedTypeHerarchyDiagnosticSuppression;
 [DiagnosticAnalyzer(LanguageNames.CSharp)]
 public sealed class SwitchExpressionSuppressor : DiagnosticSuppressor
 {
+    readonly bool _forceAllowRecords;
+
+    public SwitchExpressionSuppressor() : this(false) { }
+    
+    public SwitchExpressionSuppressor(bool forceAllowRecords)
```

Comment:
Please make this ctor `internal` and add a comment to signify that it is only intended for test purposes.
Add `InternalsVisibleTo` to the project accordingly.

Reply:
Do you prefer `InternalsVisibleTo` to be defined in the `csproj` or in a `cs` file?

Reply:
I chose the `csproj` option. Let me know if you prefer otherwise.